src/pages/pot/[potId]/payouts.tsx

```diff
@@ -318,121 +148,132 @@ export default function PayoutsTab() {
-                  const donationsForProject = allDonations.filter(
-                    (donation) => donation.recipient?.id === project_id,
-                  );
-
-                  const uniqueDonors: Record<string, any> = {};
-
-                  donationsForProject.forEach((donation) => {
-                    if (!uniqueDonors[donation.donor.id]) {
-                      uniqueDonors[donation.donor.id] = true;
-                    }
-                  });
-
-                  const donorCount = Object.keys(uniqueDonors).length;
+                </div>
+                <div className="flex w-full flex-col flex-nowrap items-center overflow-x-auto ">
+                  <div className="flex w-full flex-row items-center justify-between gap-8  bg-[#f6f6f7] p-2.5 px-4">
+                    <div className="justify-left flex w-28 flex-1 flex-row items-center">
+                      <div className="break-words  text-sm font-semibold leading-6 text-[#7B7B7B]">
+                        PROJECTS
+                      </div>
+                    </div>
+
+                    <div className="flex flex-row items-center justify-start">
+                      <div className="break-words text-sm font-semibold leading-6 text-[#7B7B7B]">
+                        POOL ALLOCATION
+                      </div>
+                    </div>
+                  </div>
+
+                  {isPayoutsPending ? (
+                    Array.from({ length: 10 }).map((_, index) => (
+                      <div key={index} className="mt-3 w-full">
+                        <Skeleton className="h-10 w-full" />
+                      </div>
+                    ))
+                  ) : payouts?.length === 0 ? (
+                    <div
+                      className="relative flex w-full flex-row items-center justify-between gap-8 p-4 md:flex-wrap md:gap-2"
+                      style={{ padding: "12px" }}
+                    >
+                      No payouts to display
+                    </div>
+                  ) : (
+                    payouts?.map((payout, index) => {
+                      const { project_id, amount } = payout;
+
+                      return (
+                        <div
+                          className="relative flex w-full flex-row items-center justify-between gap-8 p-4 md:flex-wrap md:gap-2"
+                          key={index}
+                        >
+                          <div className="flex w-[110px] flex-1 flex-row items-center justify-start gap-4 transition duration-200 hover:no-underline">
+                            <AccountProfilePicture
+                              accountId={project_id}
+                              className="h-[24px] w-[24px]"
+                            />
+                            <a
+                              className="font-semibold text-gray-800 no-underline transition duration-200 hover:text-red-600"
+                              href={`?tab=project&projectId=${project_id}`}
```

Comment:
You probably meant this?

```suggestion
                              href={`${rootHrefs.PROFILE}/${project_id}`}
```